pkgs/development/interpreters/dart/default.nix

```
@@ -51,7 +51,7 @@ stdenv.mkDerivation {
     mkdir -p $out
     cp -R * $out/
     echo $libPath
-  '' + lib.optionalString(stdenv.isLinux) ''
+  '' + lib.optionalString (stdenv.isLinux) ''
```

Formatting changes, even if "correct", ideally should be in their own commits.

```suggestion
  '' + lib.optionalString(stdenv.isLinux) ''
```